test/integration/integration_test.go

```diff
@@ -147,29 +147,29 @@ func FeatureContext(s *godog.Suite) {
 	minishift = &Minishift{runner: runner}
 
 	// steps to execute `minishift` commands
-	s.Step(`Minishift (?:has|should have) state "([^"]*)"$`,
+	s.Step(`^Minishift (?:has|should have) state "(Does Not Exist|Running|Stopped)"$`,
```

Comment:
but wrong, as it covers only the wanted states:

https://github.com/docker/machine/blob/master/libmachine/state/state.go

`Paused`, `Error` and `Timeout` are states I have also encountered during testing.